.github/workflows/cont_integration.yml

```diff
@@ -14,21 +14,9 @@ jobs:
             clippy: true
           - version: 1.56.1 # MSRV
         features:
-          - default
-          - minimal
-          - all-keys
-          - minimal,use-esplora-blocking
-          - key-value-db
-          - electrum
-          - compact_filters
-          - use-esplora-blocking,key-value-db,electrum
-          - compiler
-          - rpc
-          - verify
-          - async-interface
-          - use-esplora-async
-          - sqlite
-          - sqlite-bundled
+          - --no-default-features
+          - --all-features
+          - --features=default
```

Comment:
You should modify the build, clippy, test steps below:

Instead of `cargo build --features ${{ matrix.features }} --no-default-features`, it should be `cargo build ${{ matrix.features }}`

Reply:
oops yep forgot to do that thanks. Looks like @evanlinjin has done this.